Tests/ParsingTests/NotTests.swift

```diff
@@ -36,7 +36,7 @@ class NotTests: XCTestCase {
     XCTAssertEqual(
       input,
       """
-      // a comment
+       a comment
```

Comment:
did you mean to make this change?

Reply:
Yeah this tests that `Not` is consuming.